src/openforms/prefill/contrib/kvk/plugin.py

```diff
@@ -46,13 +46,12 @@ def get_prefill_values(
         except (RequestException, ClientError, KVKClientError):
             return {}
 
-        items = results["data"]["items"]
+        items = results["resultaten"]
         if not items:
             return {}
 
         if len(items) > 1:
-            logger.warning("multiple results for")
-            return {}
+            logger.warning("multiple results for KvK search")
 
         data = items[0]
```

Comment:
can we filter the items based on `type: hoofdvestiging` rather than just taking the first one?

Reply:
Sure, makes sense.